backend/stakepoold/config.go

```diff
@@ -80,6 +80,9 @@ type config struct {
 	RPCListeners     []string `long:"rpclisten" description:"Add an interface/port to listen for RPC connections (default port: 9113, testnet: 19113)"`
 	RPCCert          string   `long:"rpccert" description:"File containing the certificate file"`
 	RPCKey           string   `long:"rpckey" description:"File containing the certificate key"`
+	Prometheus       bool     `long:"prometheus" description:"Export metrics to be scraped by Prometheus"`
+	PrometheusWait   string   `long:"prometheuswait" description:"Amount of time wait between Prometheus metric updates (default: 1m)"`
```

Comment:
This needs a small explanation of how to input the times. Also perhaps the type should be time.Duration. For comparison https://github.com/decred/dcrd/blob/29064448cb2e3a352ad0089b56a0b747725d0ac0/config.go#L108

Reply:
Completely agree with this. I had no idea you could pass a `time.Duration` through a cli flag, thats really cool!

Reply:
Really good idea, thanks!